test/kuka_kr6_test.cpp

```diff
@@ -0,0 +1,116 @@
+#include <array>
+#include <gtest/gtest.h>
+#include <random>
+
+#include "opw_kinematics/opw_kinematics.h"
+#include "opw_kinematics/opw_parameters_examples.h" // for makeIrb2400_10<double>()
+#include "opw_kinematics/opw_utilities.h"           // for optional checking
+
+const double TOLERANCE_DOUBLE = 1e-10;
+const float  TOLERANCE_FLOAT  = 1e-5;
```

Comment:
Probably this can be achieved with templates, but I'm not sure if it makes sense to make it more complicated for such a simple concept.